recipes-ni/ni-hw-support/ni-hw-support-common_1.0.bb

```diff
@@ -0,0 +1,54 @@
+SUMMARY = "Support scripts and utilities common to all NI hardware products"
+DESCRIPTION = "Support scripts and utilities for all NI hardware products which are supported by NI LinuxRT."
+LIC_FILES_CHKSUM = "file://${COMMON_LICENSE_DIR}/MIT;md5=0835ade698e0bcf8506ecda2f7b4f302"
+LICENSE = "MIT"
+SECTION = "base"
+
+DEPENDS += "\
+	update-rc.d-native \
+"
+
+SRC_URI += "\
+	file://init.d/ni-rename-ifaces \
+"
+
+S = "${WORKDIR}"
+
+
+do_install () {
+	install -d ${D}${sysconfdir}/init.d/
+
+	install -m 0755 ${S}/init.d/ni-rename-ifaces ${D}${sysconfdir}/init.d/ni-rename-ifaces
+}
+
+pkg_postinst_${PN} () {
+	if [ -n "$D" ]; then
+		OPT="-r $D"
+	else
+		OPT="-s"
```

Comment:
What does -s do? Searching through other update-rc.d calls I don't see us using that option. Is there a reason we only use -s in the else case here?

Reply:
`-s` invokes the initscript at install-time, if the current runlevel is appropriate for it to have been executed. eg. If the script is supposed to be installed at runlevel 5, and you're installing it from RL 5, then the script will get invoked when the postinst is executed.

http://www.embeddedlinux.org.cn/OEManual/update-rc-d_class.html